modules/standard/Random.enc

```diff
@@ -37,4 +37,9 @@ read class Random
 
     END
   end
```

Comment:
This code was translated automatically at some point, and the pretty-printer adds an extra newline before an `END` (for some reason)